src/pages/Auth/Login.jsx

```diff
@@ -1,11 +1,30 @@
 import React from 'react';
 import { Footer, Navbar } from '../../components';
 
+import './auth.css';
 const Login = () => {
   return (
     <div>
       <Navbar />
-      <div className="center-items-container">Login</div>
+      <div className="center-items-container">
+        <div className="single-function-page-container">
+          <div className="single-page-heading">Login</div>
+          <hr />
+          <div className="inputs-container">
+            <label htmlFor="email">Email </label>
+            <input type="email" placeholder="adarshbalika@gmail.com" />
+            <label htmlFor="password">Password </label>
+            <input type="password" placeholder="adarshBalika123" />
```

Comment:
One suggestion, whenever you are using input fields or creating some type for form, then using the form element is a good practice.

Reply:
I agree, I will change this in upcoming prs.